lib/private/DB/QueryBuilder/ExpressionBuilder/ExpressionBuilder.php

```diff
@@ -261,7 +262,7 @@ public function gte($x, $y, $type = null) {
 	 *
 	 * @return string
 	 */
-	public function isNull($x) {
+	public function isNull(string $x): string {
```

Comment:
```suggestion
	public function isNull($x): string {
```

`$x` can be `string|ILiteral|IParameter|IQueryFunction`

Reply:
Why do all the freaking doc blocks lie?